app/models/merge_request_diff.rb

```diff
@@ -47,6 +47,23 @@ def diffs
     @diffs ||= (load_diffs(st_diffs) || [])
   end
 
+  def diffs_no_whitespace
+    # Update ref for merge request
+    merge_request.fetch_ref
```

Comment:
I think it is necessary.
`RefreshService` is refreshing all MR related with new pushed branch, even from forked projects.
But I want to update only this view's diff.